### PR TITLE
Remove flaky marker on some tests.

### PR DESCRIPTION
**Pull request by tvolkert**
They've been mostly green since https://github.com/flutter/flutter/pull/17812

### CODE REVIEW

**Review by yjbanov — APPROVED** [review]

**Comment by yjbanov** [comment]
lgtm

File: dev/devicelab/manifest.yaml [hunk 1]
```diff
@@ -199,7 +199,6 @@ tasks:
       Runs end-to-end Flutter tests on Android.
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
-    flaky: true
```
Comment (cbracken) [inline dev/devicelab/manifest.yaml]: Note for future archaeologists: the flakiness appears to have ended coincidentally with the re-imaging of all the devicelab bots. I *suspect* this is still flaky and/or dependent on a race condition.

Comment (cbracken) [reply dev/devicelab/manifest.yaml]: Though looking at that commit - I could very much believe that it's also related to the service event handling fix.

Comment (tvolkert) [reply dev/devicelab/manifest.yaml]: Three of these were definitely the service event handling fix.  One was already green in the whole dashboard.

**Pull request merged**